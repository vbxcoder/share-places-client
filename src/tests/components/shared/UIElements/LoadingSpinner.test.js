import React from 'react'
import { shallow } from 'enzyme'

import LoadingSpinner from  '../../../../shared/components/UIElements/LoadingSpinner'

test('Should render LoadingSpinner as overlay correctly', () => {
  const wrapper = shallow(<LoadingSpinner asOverlay />)
  expect(wrapper).toMatchSnapshot()
})

test('Should render LoadingSpinner without overlay correctly', () => {
  const wrapper = shallow(<LoadingSpinner />)
  expect(wrapper).toMatchSnapshot()
})