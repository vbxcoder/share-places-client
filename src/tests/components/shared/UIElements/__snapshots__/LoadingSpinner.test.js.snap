// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render LoadingSpinner as overlay correctly 1`] = `
<div
  className="loading-spinner__overlay"
>
  <div
    className="lds-dual-ring"
  />
</div>
`;

exports[`Should render LoadingSpinner without overlay correctly 1`] = `
<div
  className="undefined"
>
  <div
    className="lds-dual-ring"
  />
</div>
`;
